### ISSUE TITLE
Add <tfoot> functionality to our grid

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#1852]
### Description

Some tables are using the last row as summary. This should be coded as a `tfoot`. Is this something we can do automaticly our should the app developer set the footer in some way.
